src/shared/discovery.js

```diff
@@ -84,7 +85,14 @@ export default class Discovery {
 
     // Listen for messages from other frames.
     this._listeners.add(this.target, 'message', this._onMessage);
-    this._beacon();
+
+    // Ping specified other frames to tell them about the existence of this frame.
+    const beaconMessage = this.server
+      ? '__cross_frame_dhcp_offer'
```

Comment:
Feel free to tell me to take a hike because this couldn't be more tangential, but I'm curious as to the meaning of `dhcp` in this context. Is it meaningful? DCHP as I (don't really) understand it seems pretty specific. Again, just a curiosity question.